tests/core/specific_heat_phase_change_01.output

```diff
@@ -0,0 +1,37 @@
+
+DEAL::Beggining
+DEAL::Testing solid fraction xi
+DEAL:: T = 0.5  , xi = 0.00000
+DEAL:: T = 1    , xi = 0.00000
+DEAL:: T = 1.5  , xi = 0.500000
+DEAL:: T = 1.75 , xi = 0.750000
+DEAL:: T = 2    , xi = 1.00000
+DEAL:: T = 2.5  , xi = 1.00000
+DEAL::Testing enthalpy H
+DEAL:: T = 0.5  , H = 0.500000
+DEAL:: T = 1    , H = 1.00000
+DEAL:: T = 1.5  , H = 52.0000
+DEAL:: T = 2    , H = 103.000
+DEAL:: T = 2.5  , H = 104.500
+DEAL::Testing specific heat
+DEAL:: T_0 = 0.100000 T_1 = 0.300000 Cp = 1.00000
+DEAL:: T_0 = 0.300000 T_1 = 0.500000 Cp = 1.00000
+DEAL:: T_0 = 0.500000 T_1 = 0.700000 Cp = 1.00000
+DEAL:: T_0 = 0.700000 T_1 = 0.900000 Cp = 1.00000
+DEAL:: T_0 = 0.900000 T_1 = 1.10000 Cp = 51.5000
+DEAL:: T_0 = 1.10000 T_1 = 1.30000 Cp = 102.000
+DEAL:: T_0 = 1.30000 T_1 = 1.50000 Cp = 102.000
+DEAL:: T_0 = 1.50000 T_1 = 1.70000 Cp = 102.000
```

Comment:
why cp is increasing from 1 to 102? is this the artificial specific heat to model the phase change? if yes, I think it would better to explain this in a sentence in the test header.

Reply:
Yup that's exactly it. Specific heat increases in the phase change buffer zone then reverts to a lower value.  I'll add a better comment in the test and documentation.

Reply:
I improved the message and also changed the message for the other tests.